src/value/owned.rs

```diff
@@ -71,20 +71,17 @@ impl OwnedSymbolToken {
         }
     }
 
-    /// Decorates the [`OwnedSymbolToken`] with text.
-    pub fn with_text<T: Into<Rc<str>>>(mut self, text: T) -> Self {
-        self.text = Some(text.into());
+    fn with_text_owned(mut self, text: &'static str) -> Self {
+        self.text = Some(Rc::from(text));
         self
     }
 
-    /// Decorates the [`OwnedSymbolToken`] with a local ID.
-    pub fn with_local_sid(mut self, local_sid: SymbolId) -> Self {
+    fn with_local_sid_owned(mut self, local_sid: SymbolId) -> Self {
         self.local_sid = Some(local_sid);
         self
     }
 
-    /// Decorates the [`OwnedSymbolToken`] with an [`OwnedImportSource`].
-    pub fn with_source<T: Into<Rc<str>>>(mut self, table: T, sid: SymbolId) -> Self {
+    fn with_source_owned(mut self, table: &'static str, sid: SymbolId) -> Self {
         self.source = Some(OwnedImportSource::new(table, sid));
```

Comment:
Same question on this API, why do we need `..._owned` variants when they look the same as the non-suffix ones below?